reo/src/utils.jl

```diff
@@ -104,49 +105,50 @@ Base.@kwdef struct Parameter
      MaxDemandInTier::Array{Float64,1}  # \delta^{t}_{e}: Maximum power demand in ratchet e
      MaxDemandMonthsInTier::Array{Float64,1}   # \delta^{mt}_{n}: Maximum monthly power demand in tier n
      MaxUsageInTier::Array{Float64,1}   # \delta^{tu}_{u}: Maximum monthly energy demand in tier u
-	 
-	 
+
+
 	 ###  Incentive Parameters ###
 	 NMILLimits::AxisArray   # i^{n}_{v}: Net metering and interconnect limits in net metering regime v [kW]
      MaxProdIncent::AxisArray      # \bar{i}_t: Upper incentive limit for technology t [$]
 	 ProductionIncentiveRate::AxisArray  # i^{r}_{t}: Incentive rate for technology t [$/kWh] (NEW)
-	 MaxSizeForProdIncent::AxisArray  # \bar{i}^{\sigma}_t: Maximum system size to obtain production incentive for technology t [kW]	 
-	 
+	 MaxSizeForProdIncent::AxisArray  # \bar{i}^{\sigma}_t: Maximum system size to obtain production incentive for technology t [kW]
+
 	 ###  Technology-specific Time-series Factor Parameters ###
 	 ProductionFactor::AxisArray    #f^{p}_{th}  Production factor of technology t and time step h  [unitless]  (NEW)
-     # f^{fa}_{th}: Fuel burn ambient correction factor of technology t at time step h [unitless] 
-	 # f^{ha}_{th}: Hot water ambient correction factor of technology t at time step h [unitless] 
-	 # f^{ht}_{th}: Hot water thermal grade correction factor t correction factor of technology t at time step h [unitless] 
-	 # f^{ed}_{th}: Fuel burn ambient correction factor of technology t at time step h [unitless] 
-	 
+     #CHPThermalProdSlope # f^{fa}_{th}: Fuel burn ambient correction factor of technology t at time step h [unitless]
```

Comment:
@Bill-Becker there are a lot of commented out lines in utils.jl that look like we can delete them. Can you please go through it and remove all the obsolete stuff? (I don't want to accidentally remove something that you are saving for CHP).

Reply:
Removed this in 19a830492c9101dd3917ee094bf9e33b70e499c7